### PR TITLE
a beautiful wall clock

### PR DESCRIPTION
**Pull request by Deepak8717**
# Description

A new project called analog clock has been added. its a beautiful wall clock

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ -] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [ -] I have performed a self-review of my own code
- [ -] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by atapas** [comment]
@Deepak8717 Please create an issue and associate it.

File: src/plays/analog-clock/AnalogClock.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    const interval = setInterval(() => {
+      setDate(new Date());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  });
+  const hour = date.getHours() * 30 + 180; // 360/12 = 30 each hour it rotates 30deg +180 (initial design at 0 deg was upside down so to fix it i rotate 180deg..// see the design at 0deg for more clarification)
+  const minute = date.getMinutes() * 6 + 180; // 360/60 = 6 each minutes it rotate 6deg + 180(same as above to ...see at 0 deg)
+  const second = date.getSeconds() * 6 + 180; //// 360/60 = 6 each minutes it rotate 6deg + 180(look at 0 deg for more clarification)
+  console.log(date.getHours());
```
Comment (atapas) [inline src/plays/analog-clock/AnalogClock.jsx:26]: Remove console log

File: src/plays/analog-clock/AnalogClock.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  // Your Code Start below.
+  const [date, setDate] = useState(new Date());
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setDate(new Date());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  });
```
Comment (atapas) [inline src/plays/analog-clock/AnalogClock.jsx:22]: The useEffect should not run everytime.

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by Deepak8717** [comment]
@atapas done.

**Review by atapas — APPROVED** [review]

**Pull request merged**